src/morse/morse.ts

```diff
@@ -355,8 +355,10 @@ export class MorseViewModel {
       this.runningPlayMs(0)
       // clear the voice cache
       this.voiceBuffer = []
-      // prime the pump for safari
-      this.morseVoice.primeThePump()
+      if (this.morseVoice.voiceEnabled() && navigator.userAgent.indexOf('CrOS') === -1) {
+        // prime the pump for safari
```

Comment:
Nitpick, should this comment change to reflect what is happening? Or if the comment is accurate, should the pump only be primed if the browser is safari?